src/map/script.c

```diff
@@ -12749,19 +12749,23 @@ static BUILDIN(globalmes)
 
 /// Creates a waiting room (chat room) for this npc.
 ///
-/// waitingroom "<title>",<limit>{,"<event>"{,<trigger>{,<zeny>{,<minlvl>{,<maxlvl>}}}}};
+/// waitingroom "<title>",<limit>{,"<event>"{,<trigger>{,<zeny>{,<minlvl>{,<maxlvl>{,<npcname>}}}}}};
 static BUILDIN(waitingroom)
 {
-	struct npc_data* nd;
-	const char* title = script_getstr(st, 2);
+	struct npc_data *nd;
+	const char *title = script_getstr(st, 2);
 	int limit = script_getnum(st, 3);
-	const char* ev = script_hasdata(st,4) ? script_getstr(st,4) : "";
-	int trigger =  script_hasdata(st,5) ? script_getnum(st,5) : limit;
-	int zeny =  script_hasdata(st,6) ? script_getnum(st,6) : 0;
-	int minLvl =  script_hasdata(st,7) ? script_getnum(st,7) : 1;
-	int maxLvl =  script_hasdata(st,8) ? script_getnum(st,8) : MAX_LEVEL;
+	const char *ev = script_hasdata(st, 4) ? script_getstr(st, 4) : "";
+	int trigger = script_hasdata(st, 5) ? script_getnum(st, 5) : limit;
+	int zeny = script_hasdata(st, 6) ? script_getnum(st, 6) : 0;
+	int minLvl = script_hasdata(st, 7) ? script_getnum(st, 7) : 1;
+	int maxLvl = script_hasdata(st, 8) ? script_getnum(st, 8) : MAX_LEVEL;
 
+	if (script_hasdata(st, 9))
+		nd = npc->name2id(script_getstr(st, 9));
+	else
 	nd = map->id2nd(st->oid);
```

Comment:
please separate your white space edit and actual edit into separate commit
example this line should also have an extra tab

Reply:
imo, that look unnecessary, and look extra/irrelevant in the history.
could have just do a huge commit that fix all white space edit if they ever wanted that.

Reply:
https://github.com/HerculesWS/Hercules/pull/2080#discussion_r262317146 <- *still don't have time to update it*
I bet you haven't seen Jedzkie big commit before
the commit so large that some lines has insert discreetly, need a sharp eye to detect it

Reply:
if you're referring tot those whitespace that exists for other script commands, i have removed it, no idea why it was edited in the first place, probably the visual studio auto adjusted it during file parsing time.